OpenSim/Common/DataTable.h

```diff
@@ -721,6 +721,56 @@ class DataTable_ : public AbstractDataTable {
         return _depData.updRow((int)std::distance(_indData.cbegin(), iter));
     }
 
+    /** Set row at index. Equivalent to
+    ```
+    updRowAtIndex(index) = depRow;
+    ```
+
+    \throws RowIndexOutOfRange If the index is out of range.                  */
+    void setRowAtIndex(size_t index, const RowVectorView& depRow) {
```

Comment:
Minor: I suggest adjusting the ordering of these methods. They're currently
- 00 setRowAtIndex with RowVectorView
- 11 setRow with RowVector
- 01 setRowAtIndex with RowVector
- 10 setRow with RowVectorView

I suggest either 00-01-10-11 or 00-10-01-11.